configs/pie-components-config/playwright-lit-visual-config.js

```diff
@@ -51,7 +51,7 @@ export function getPlaywrightVisualConfig () {
                 name: 'visual:mobile',
                 grep: /@mobile/,
                 use: {
-                    ...devices['Pixel 5'],
+                    ...devices['Galaxy S8'],
```

Comment:
This only applies to tests with `@mobile` in their name. Currently we don't have any in the components folder so this won't affect any other tests.

The reason I'm adding it is that there is a bit of a problem in how Percy works, where it the HTML snapshot is rendered at a desktop width and sent to the server and re-rendered at a mobile width, where the textarea should be taller. By telling the test to use a narrower viewport it means the initial snapshot more closely matches the final screenshot.